libs/sdk-model/src/dashboard/widget.ts

```diff
@@ -23,40 +22,17 @@ export type IWidgetDefinition = IKpiWidgetDefinition | IInsightWidgetDefinition;
  */
 export type IWidget = IKpiWidget | IInsightWidget;
 
-/**
- * List of currently supported types of references that can be retrieved using getWidgetReferencedObjects()
- * @alpha
- */
-export type SupportedWidgetReferenceTypes = Exclude<
```

Comment:
Where was this moved to?

Reply:
It's collocated with IWorkspaceDashboardsService. I'm not sure if it should be part of sdk-model :/